yscope-k8s/README.md

```
@@ -0,0 +1,107 @@
+Helm is the package manager for Kubernetes.
+
+Follow the guide here: [helm]
+
+# Launch clp-package
+1. Find the clp-package for test on our official website [clp-json-v0.4.0]. We also put the dataset for demo here: `mongod-256MB-presto-clp.log.tar.gz`.
+
+2. Untar it.
+
+3. Replace the content of `/path/to/clp-json-package/etc/clp-config.yml` with the output of `demo-assets/init.sh <ip_addr>` where the `<ip_addr>` is the IP address of the host that you are running the clp-package.
+
+4. Launch:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)